lighthouse-core/config/v2/default.json

```diff
@@ -0,0 +1,304 @@
+{
+  "configVersion": 2,
+
+  "gatherers": {
```

Comment:
yep :) lists all gatherers with their ids and paths (if different from id) plus allowing options and multiple version of the same implementation later on down the road